packages/database/migration/20250722075349_sunset_weekly_summary/migration.ts

```diff
@@ -0,0 +1,14 @@
+import type { MigrationScript } from "../../src/scripts/migration-runner";
+
+export const sunsetWeeklySummary: MigrationScript = {
```

Comment:
could you please also add a comment here about what the migration script does? would be helpful if we take a look at it again 🙏